feature_embeddings.py

```diff
@@ -0,0 +1,149 @@
+"""
+Words are split on part-word things and the labels of these part words are
+duplicated (bert pre-processing)
+
+Creates input_ids, tags, and masks
+todo: option that pre-processes lexicals too
+
+Code adapted from
+https://www.depends-on-the-definition.com/named-entity-recognition-with-bert/
+and
+https://github.com/abhishekkrthakur/bert-entity-extraction
+
+"""
+from transformers import BertTokenizer
+from torch.utils.data import TensorDataset
+import pandas as pd
+import torch
+
+class SentenceGetter(object):
+
+    def __init__(self, data):
+        self.n_sent = 1
+        self.data = data
+        self.empty = False
+        # todo: add other lexical features (w, t, pos, lemma, stem)
+        agg_func = lambda s: [(w, t) for w, t in zip(s["token"].values.tolist(),

```

Comment:
This part seems needlessly complicated in combination with the multiple times labels and their tags are zipped and unzipped again (starting from line 87), but if it ain't broke...